include/ofi_mr.h

```diff
@@ -318,6 +318,13 @@ bool ofi_mr_cache_flush(struct ofi_mr_cache *cache, bool flush_lru);
 
 int ofi_mr_cache_search(struct ofi_mr_cache *cache, const struct fi_mr_attr *attr,
 			struct ofi_mr_entry **entry);
+
+static inline bool ofi_mr_cache_iface_enabled(struct ofi_mr_cache *cache,
+					      enum fi_hmem_iface iface)
+{
+	return cache->monitors[iface] != NULL;
+}
```

Comment:
Since there's no user of this function, it's hard to know if the expected use case makes sense.

Reply:
I'll remove for now. We can revisit later.